packages/asset-swapper/src/utils/rfq_maker_blacklist.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Tracks a maker's history of timely responses, and manages whether a given
+ * maker should be avoided for being too latent.
+ */
+export class RfqMakerBlacklist {
+    // tslint:disable-next-line:custom-no-magic-numbers
+    private static readonly _msPerMinute = 1000 * 60;
```

Comment:
Can you replace `1000` with `ONE_SECOND`?

Reply:
Should this live in this class? or could it be in the same file of `ONE_SECOND` or another const file?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2706/commits/79e0a9ef3701f8bdb2e28dd4a6b83bce88019472